bootrom/Makefile

```diff
@@ -3,20 +3,12 @@
 
 all: build
 
-PY := python3
```

Comment:
I had these to reflect consistency between the Makefiles. From what I've seen, this is also the norm.

Reply:
Technically the only python is python3 now ;)